test/unit/linear_algebra/test_lapacke.cpp

```diff
@@ -0,0 +1,138 @@
+#include <tuvx/linear_algebra/linear_algebra.hpp>
+
+#include <gtest/gtest.h>
+
+#include <cfloat>
+#include <chrono>
+#include <cstddef>
+#include <cstdlib>
+#include <cstring>
+#include <ctime>
+#include <fstream>
+#include <lapacke.h>
+#include <limits>
+#include <vector>
+
+using namespace tuvx;
+using std::chrono::duration;
+using std::chrono::duration_cast;
+using std::chrono::high_resolution_clock;
+using std::chrono::milliseconds;
```

Comment:
@mattldawson @K20shores is there an agreement whether we do `using` here or spell them out wherever they are used?

Reply:
If it's in tests we can use using. My bigger concern is that we seem to be recreating google benchmark. I feel like it's better to keep all benchmarking stuff separate form the unit tests.

Reply:
@K20shores I have separated the tests and benchmarks in my latest commit (also, switched to google benchmark from std::chrono)